recipes/argoverse/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set name = "argoverse" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/argoai/{{ name }}-api/archive/master.tar.gz
+  sha256: 8f46ba59cd28128c3a2ba35111b855e826c66e2911ea1ac779590665d6db897c
+
+build:
+  number: 0
+  skip: true                                 # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python {{ python }}                    # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - pybind11                               # [build_platform != target_platform]
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pip
+    - colour
+    - descartes
+    - imageio
+    - h5py
+    - hydra-core
+    - lap
+    - matplotlib
```

https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib

```suggestion
    - matplotlib-base
```